scripts/docker-build.sh

```diff
@@ -5,16 +5,16 @@ DOCKER_REPOSITORY="${DOCKER_REPOSITORY:-quay.io/unstructured-io/unstructured}"
 PIP_VERSION="${PIP_VERSION:-23.1.2}"
 DOCKER_IMAGE="${DOCKER_IMAGE:-unstructured:dev}"
 
-DOCKER_BUILD_CMD=(docker buildx build --load -f Dockerfile \
-  --build-arg PIP_VERSION="$PIP_VERSION" \
-  --build-arg BUILDKIT_INLINE_CACHE=1 \
-  --progress plain \
-  --cache-from "$DOCKER_REPOSITORY":latest \
-  -t "$DOCKER_IMAGE" .)
+DOCKER_BUILD_CMD=(docker buildx build --load -f Dockerfile
```

Comment:
Double check that backslashes aren't needed. :)

Reply:
Verified, this works when building the docker image